src/ProjectTemplates/README.md

```diff
@@ -10,3 +10,55 @@ To update project template JavaScript dependencies:
     ```
 
 To add a new dependency, run `npm install <package-name>` and update the `scripts` section in `package.json` to specify how the new dependency should be copied into its template.
+
+# Testing AI templates locally
+
+Regardless of how you run AI templates locally, you must first generate NuGet packages for `Microsoft.Extensions.AI` projects defined within this repo:
+```sh
+.\build.cmd -vs AI -noLaunch # Generate an SDK.sln for Microsoft.Extensions.AI* projects
+.\build.cmd -build -pack     # Build a NuGet package for each project
+```
+
+**Note:** Since package versions don't change between local builds, it may be necessary to occasionally delete `Microsoft.Extensions.AI*` packages from your local nuget cache, especially if you're making changes to these packages. An example of how to do this in PowerShell is:
+```pwsh
+Remove-Item ~\.nuget\packages\microsoft.extensions.ai* -Recurse -Force
+```
+
+## Running directly within the repo
+
+Navigate to the `Microsoft.Extensions.AI.Templates` folder and run:
+```sh
+dotnet build
+```
```

Comment:
I don't think this will work directly. In order to use "dotnet" within the repo, the SDK version declared in the global.json must either be installed globally, or the path to the repo's SDK must be prepended to the %PATH%.

Reply:
Thanks, updated the build instructions to be more specific.